src/ripple/test/jtx/JTx.h

```diff
@@ -56,12 +59,99 @@ struct JTx
     {
     }
 
+#if defined(_MSC_VER) && _MSC_VER <= 1800
```

Comment:
With the suggestion: if the parameter is an rvalue ref, I do two move constructs (vs. one move construct with the current). If the parameter is an lvalue ref, I do a copy construct and a move construct (vs a copy construct for the current). We pay an extra move construct. *If* move constructs are cheap (and I need to check that it is), then it's a small price to pay for one function instead of two.